builder/builder-next/exporter/wrapper.go

```diff
@@ -47,7 +57,9 @@ func (e *imageExporterMobyWrapper) Resolve(ctx context.Context, id int, exporter
 	if err != nil {
 		return nil, err
 	}
-	exporterAttrs[string(exptypes.OptKeyName)] = strings.Join(reposAndTags, ",")
+
+	// Force the exporter to not use a name so it always creates a dangling image.
+	exporterAttrs[string(exptypes.OptKeyName)] = ""
```

Comment:
Since name will always be empty now - can we make use of the `OptKeyDanglingEmptyOnly` now?

Reply:
What do you mean? We can potentially remove it since name will always be empty but it also doesn't harm anything to keep it.

Reply:
Yeah I meant not forcing `OptKeyDanglingEmptyOnly = true` (the default is false).

It doesn't harm, but if the buildkit default settings make sense, we should try to stick to them to make the wrapper as minimal as possible.

Reply:
If it's ok, I can leave this for cleanup. CI is passing and I'd prefer not to trigger CI again.